client/payment-details/dispute-details/dispute-awaiting-response-details.tsx

```diff
@@ -255,9 +255,10 @@ const DisputeAwaitingResponseDetails: React.FC< Props > = ( {
 									onClick={ () => {
 										wcpayTracks.recordEvent(
 											wcpayTracks.events
-												.DISPUTE_CHALLENGE_CLICK,
+												.DISPUTE_CHALLENGE_CLICKED,
 											{
 												dispute_status: dispute.status,
```

Comment:
I realise that `dispute_status` is also redundant, since this should only every be `needs_response` (inquiries have a different event, `wcpay_dispute_inquiry_refund_click`)